lib/templates/deploy.batch.go.template

```diff
@@ -22,8 +13,23 @@ IF /I "%IN_PLACE_DEPLOYMENT%" EQU "1" (
 
 ECHO Prepare workspace
 IF "%GOROOT%" == "" (
-    :: select go version TODO
-    SET GOROOT=D:\Program Files\go\1.4.2
+  IF DEFINED GO_WEB_CONFIG_TEMPLATE (
+    :: Sort folder in reverse alphabetically order (-N), and folder only (G)
```

Comment:
This is probably not correct, as you'd end up with 1.4.2 > 1.4.10. Doing correct SemVer comparison can be challenging.